ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -168,50 +164,69 @@
        :collision (format "Subscription with with provider-id [%s] and native-id [%s] already exists."
                           provider-id
                           native-id)))
-    (let [concept (api-core/body->concept!
-                   :subscription
-                   provider-id
-                   native-id
-                   body
-                   content-type
-                   headers)]
-      (check-subscription-ingest-permission request-context concept provider-id)
-      (perform-subscription-ingest request-context concept headers))))
+    (let [new-subscription (api-core/body->concept!
+                            :subscription
+                            provider-id
+                            native-id
+                            body
+                            content-type
+                            headers)
+          subscriber-id (:SubscriberId (json/decode (:metadata new-subscription) true))]
+      (check-subscription-management-acls request-context subscriber-id provider-id)
+      (perform-subscription-ingest request-context new-subscription headers))))
 
 (defn create-or-update-subscription-with-native-id
   "Processes a request to create or update a subscription. This function
-  does NOT check for collisions. This is mapped to PUT methods to preserve
+  does NOT fail on collisions. This is mapped to PUT methods to preserve
   existing functionality."
   [provider-id native-id request]
-  (let [{:keys [body content-type headers request-context]} request]
+  (let [{:keys [body content-type headers request-context]} request
+        new-subscription (api-core/body->concept! :subscription
```

Comment:
There is improvement that can be done in this area of the code. We can do that later.